state/allwatcher.go

```diff
@@ -1899,8 +1902,10 @@ type allWatcherContext struct {
 	statuses    map[string]status.StatusInfo
 	instances   map[string]instanceData
 	// A map of the existing MachinePortRanges where the keys are machine IDs.
-	openPortRanges map[string]MachinePortRanges
-	userAccess     map[string]map[string]permission.Access
+	openPortRangesForMachine map[string]MachinePortRanges
+	// A map of the existing ApplicationPortRanges where the keys are application names.
+	openPortRangesForApplication map[string]ApplicationPortRanges
```

Comment:
Shouldn't this be "unit" since we're not doing application port ranges yet?
This attribute simply exists to populate the UnitInfo "OpenPortRangesByEndpoint" value. Iaas will do that its own way due to the (unnecessary) machine grouping. But for caas, we can just track the unit ports directly.